Category.m

```diff
@@ -4,27 +4,34 @@
     %   how much a given contour matches others in this category.
     
     properties
-        average  % The average of all contours in the category;
-                 % this is what prospective members are compared to
-        contours % An array of the contour objects in this category
+        reference  % The average of all contours in the category;
+                   % this is what prospective members are compared to
+        size % The number of contours in this category
     end
     
     methods
-        function cat = Category(contours)
+        function cat = Category(size, reference)
             % Construct a new category from the given array, which must
             % contain at least one contour
-            cat.contours = contours;
-            cat.average  = ContourFactory.getAverage(contours);
```

Comment:
`ContourFactory` doesn't have `.getAverage( )` yet. Are you sure we want to have that in there?